### ISSUE TITLE
After using custom filter, all other filters seem to reset

### ISSUE THREAD

**Issue by molerat619** [matfish2/vue-tables-2#481]
- Vue.js version: 2.5.13
- consumed using: Laravel

I have a button that toggles a filter that filters for active/inactive entries. For that, I have to use a requestAdapter. But as soon as I use that filter, the other filters are reset - **BUT** only in the view. If I check the network tab, the table still sends the right request to the backend. So here is some code:

Options:
```
initFilters: {
	active: true,
	start_date: {start: moment().subtract(1, 'weeks'), end: moment().add(2, 'weeks')},
	end_date: {start: moment().subtract(1, 'days'), end: moment().add(100, 'years')}
},
requestAdapter: function (data) {
	data.query.active = data.active;
	data.active = undefined;

	return data;
}
```

HTML: 
```
<div slot="filter__options">
	<button @click="toggleStornoFilter()>Stornos anzeigen</button>
</div>
```

So again, what happens is that the daterange filters for example stop showing, even though they send the right query to the backend. See these screenshots:

Before clicking "Stornos anzeigen":
![image](https://user-images.githubusercontent.com/3458576/38015037-d57f9098-326a-11e8-896a-87245012b344.png)

After clicking "Stornos anzeigen":
![image](https://user-images.githubusercontent.com/3458576/38015049-e8315f1e-326a-11e8-9d83-a97d307a70b8.png)
As you can see, the daterange filters are reset, but now check this request out:

![image](https://user-images.githubusercontent.com/3458576/38015159-3acefb82-326b-11e8-928b-eaff77174480.png)
So it is sending all the right stuff. 

Also, in this state, I can no longer click the Daterange Filters to see the picker.

**Comment by molerat619** [comment]
I think this has something to do with #484

**Comment by matfish2** [comment]
I think that was fixed by solving #496. Please check and let me know
